src/main/java/org/apache/tomcat/jni/SSL.java

```diff
@@ -237,6 +237,15 @@
     public static final int SSL_ST_CONNECT = 0x1000;
     public static final int SSL_ST_ACCEPT =  0x2000;
 
+    /**
+     * The actual release that is used and implement the OPENSSL API / ABI
+     */
+    public enum Release {
```

Comment:
@nmittler @Scottmitch not sure if Release is a good name... suggestions welcome :)